panic: runtime error: slice bounds out of range

Happens when switching /win's

```
panic: runtime error: slice bounds out of range                                   

goroutine 6 [running]:
panic(0x7c80a0, 0xc42000c140)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Display(0xc420240a00, 0x4)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:243 +0xb00
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Render(0xc420240a00)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:106 +0x53
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Visibility(0xc420240a00, 0x2, 0x1)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:82 +0x87
main.(*ZKC).focus.func1()
        /home/dhill/.go/src/github.com/companyzero/zkc/zkclient/zkclient.go:185 +0x21f
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4.func1(0xc42000c500, 0xc420054770, 0xc42000c4f0, 0xc420010320)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:283 +0xf1
created by github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:286 +0x239
```

slightly different.  race?

```
panic: runtime error: slice bounds out of range

goroutine 6 [running]:
panic(0x7c90a0, 0xc42000c140)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Display(0xc4200a4f00, 0x4)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:243 +0xb00
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Render(0xc4200a4f00)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:106 +0x53
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*Window).render(0xc420060d20)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/window.go:120 +0x71
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.resizeAndRender(0xc420060d20)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:538 +0xa1
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.initKeyHandler.func2()
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:344 +0x2d
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4.func1(0xc42000c500, 0xc420054770, 0xc42000c4f0, 0xc420010320)
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:283 +0xf1
created by github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4
        /home/dhill/.go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:286 +0x239
```

Just hit this.

```
panic: runtime error: slice bounds out of range

goroutine 6 [running]:
panic(0x7ca180, 0xc42000c110)
        /home/jrick/go/src/runtime/panic.go:500 +0x1a1
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Display(0xc4200a8d80, 0x4)
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:243 +0xb00
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*List).Render(0xc4200a8d80)
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/list.go:106 +0x53
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*Window).render(0xc42006cd20)
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/window.go:120 +0x71
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.resizeAndRender(0xc42006cd20)
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:538 +0xa1
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.initKeyHandler.func2()
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:344 +0x2d
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4.func1(0xc42000c4a0, 0xc4200147e0, 0xc42000c3f8, 0xc420010220)
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:283 +0xf1
created by github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4
        /home/jrick/src/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:286 +0x239
```

--- a/list.go
+++ b/list.go
@@ -155,7 +155,7 @@ func (l *List) Resize() {
 	}
 
 	// check if we need to update l.at
-	if l.at != 0 && l.at + l.trueH >= len(l.content) {
+	if l.at != 0 && l.at+l.trueH >= len(l.content) {
 		l.at = len(l.content) - l.trueH
 		if l.at < 0 {
 			l.at = 0

--- a/list.go
+++ b/list.go
@@ -153,6 +153,14 @@ func (l *List) Resize() {
 	if l.height < 1 {
 		l.trueH = l.w.y + l.height - 1
 	}
+
+	// check if we need to update l.at
+	if l.at != 0 && l.at + l.trueH >= len(l.content) {
+		l.at = len(l.content) - l.trueH
+		if l.at < 0 {
+			l.at = 0
+		}
+	}
 }
 
 // AddList is a convenience function to add a new list to a window.  It wraps

